CRM/Report/Form/Mailing/Summary.php

```diff
@@ -373,7 +391,15 @@ public function select() {
             }
             else {
               if (in_array($tableName, $count_tables)) {
-                $select[] = "count(DISTINCT {$field['dbAlias']}) as {$tableName}_{$fieldName}";
+                // Use the DISTINCT keyword appropriately, based on the contents
+                // of $distinct_count_columns.
+                if (in_array("{$tableName}.{$fieldName}", $distinct_count_columns)) {
+                  $distinct = 'DISTINCT';
+                }
+                else {
+                  $distinct = '';
+                }
```

Comment:
can we remove else part and initialize `$distinct` variable above if condition ?